diff --git a/node_modules/next/dist/client/image.d.ts b/node_modules/next/dist/client/image.d.ts
index 53c9486..2179e66 100644
--- a/node_modules/next/dist/client/image.d.ts
+++ b/node_modules/next/dist/client/image.d.ts
@@ -54,6 +54,6 @@ export declare type ImageProps = Omit<JSX.IntrinsicElements['img'], 'src' | 'src
     unoptimized?: boolean;
     objectFit?: ImgElementStyle['objectFit'];
     objectPosition?: ImgElementStyle['objectPosition'];
-} & (StringImageProps | ObjectImageProps);
+} & StringImageProps;
 export default function Image({ src, sizes, unoptimized, priority, loading, className, quality, width, height, objectFit, objectPosition, loader, placeholder, blurDataURL, ...all }: ImageProps): JSX.Element;
 export {};
diff --git a/node_modules/next/types/global.d.ts b/node_modules/next/types/global.d.ts
index d753639..d751400 100644
--- a/node_modules/next/types/global.d.ts
+++ b/node_modules/next/types/global.d.ts
@@ -39,12 +39,6 @@ declare module '*.png' {
   export default content
 }
 
-declare module '*.svg' {
-  const content: StaticImageData
-
-  export default content
-}
-
 declare module '*.jpg' {
   const content: StaticImageData
 
